src/main/resources/static/requestBlood.html

```diff
@@ -0,0 +1,315 @@
+   <input style="color: black;"  type="text" name="myCity" id="myCity"
+   required = "required">
+   <br>
+   
+  <label>*State:</label>
+    <select  style="color: black;" name="myState" id = "myState">
+	<option value="MI" class = "others">Michigan</option>
+    </select> 
+    <br>
+
+    <label for="zipCode">*Zip Code:</label>
+    <input style="color: black;"  type="text" name="zipCode" id="zipCode"
+           required = "required">
+    <br>
+
+   <label for="description">*Situation:</label>
+  <textarea  style="color: black;" name="Text1" cols="40" rows="5" style="color=black" id = "situation"
+             required = "required"> </textarea>
+   <br>
+
+</form>
+<button onclick = "validate()">Submit</button>
+              </div>
+              </div>
+            </div>
+        </div>
+    </div>
+</div>
+
+<script>
+
+function validate()
+{
+    if(checkForm() == true )
+    {
+        console.log("hi");
+        var firstName = document.getElementById("myFName").value;
+        var lastName = document.getElementById("myLName").value;
+        var email = document.getElementById("myEmail").value;
+        var phoneNumber = document.getElementById("myPhone").value;
+        var bloodType =  document.getElementById("bloodType").value;
+        var hospital = document.getElementById("hospital").value;
+        var street = document.getElementById("myStreet").value;
+        var city = document.getElementById("myCity").value;
+        var state = document.getElementById("myState").value;
+        var zipCode = document.getElementById("zipCode").value;
+        var situation = document.getElementById("situation").value;
+
+        var requestInfo =
+        {
+            firstName: firstName ,
+            lastName : lastName ,
+            email : email,
+            phoneNumber : phoneNumber,
+            bloodType: bloodType
+            hospitalName : hospital,
+            streetName : street,
+            cityName : city,
+            stateName : state,
+            zipCode : zipCode,
+            message : situation
+
+
+        }
+                    $.ajax(
+                    {
+                        type:"POST",
+                        url: "/requestBlood",
+                        data:JSON.stringify(requestInfo),
+                        contentType: 'application/json',
+                        dataType:"json",
+                        success: function(data){
+                            alert("Form submitted successfully!");
+                        },
+                        failure: function(error){
+                            alert("Sorry your request did not go through!");
+                        },
+                        error: function(error){
+                            alert(error);
+                        }
+                    });
+    }
+    else
+    {
+        alert("Fields cannot be empty")
+        console.log("fields are empty");
+    }
+
+}
+
+</script>
+
+
+<script>
```

Comment:
is there a reason why you made to separate script blocks?

Reply:
will fix it